component-api/src/main/java/org/talend/sdk/component/api/service/dependency/Resolver.java

```diff
@@ -82,5 +82,8 @@ interface ClassLoaderDescriptor extends AutoCloseable {
          * @return the dependencies who matched the resolution and were used to create the classloader.
          */
         Collection<String> resolvedDependencies();
+
+        default void setUseURLConnectionCaches(final boolean useURLConnectionCaches) {
```

Comment:
maybe no need to use this "default", as only ClassLoaderDescriptorImpl implement this interface?